### PR TITLE
upd(goverlay): `1.0` -> `1.1.1`

### PR DESCRIPTION
**Pull request by Xdavius**
- Goverlay is full Qt6 from upstream (Thx to Antiz)
- Add pacdeps=("qt6pas")
- Changed URL to point to Source Package (Was binary before, without -bin)
- Is now built from source
- Changed maintainer name to Xdavius

### CODE REVIEW

**Comment by Xdavius** [comment]
Need qt6pas merge before

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Please don't plop in the PKGBUILD, try to rework the pacscript as is.

**Comment by Xdavius** [comment]
> Please don't plop in the PKGBUILD, try to rework the pacscript as is.

Will see later.

Need to push again after qt6pas merge to check if some makedepends are missing too (wasn't on my side but...)

Ok for check function.

But still don't understand why cancelling pkgver line even if it's implicit. 

Please, wait a bit for this one.

I have re-uploaded you're original one as -git for now, without update, as an alternative before I test the -bin new release on my side (tempus fugit for now)

Thank you.

**Comment by Elsie19** [comment]
> But still don't understand why cancelling pkgver line even if it's implicit.

`pkgrel` defaults to `1` if not included, it only clutters it if you include it with it's default value.

File: packages/goverlay/goverlay.pacscript [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+homepage="https://github.com/benjamimgois/goverlay"
+license=('GPL-3.0-or-later')
+depends=('qt6pas' 'mangohud')
+makedepends=('git' 'lazarus-ide' 'libglu1-mesa-dev' 'qt6-base-dev')
+pacdeps=('qt6pas')
+checkdepends=('appstream' 'desktop-file-utils')
+
+build() {
+	make -j "${NCPU}" LAZBUILDOPTS=--lazarusdir=/usr/lib/lazarus/default
+}
```
[the remark this answers is not in the document]
Comment (Xdavius) [reply packages/goverlay/goverlay.pacscript:16]: > shfmt.

It dosen't change anything.
here : I see 2 spaces
When I download, see a tab with 4 spaces

When run shfmt, see 4 spaces too.

[the remark this answers is not in the document]
Comment (Xdavius) [reply packages/goverlay/goverlay.pacscript:16]: reinstalled pre-commit, now OK

**Comment by Xdavius** [comment]
Need lazarus-ide > 3.0 as pacdeps to build.

Working on lazarus-ide-qt6 to provide it.

**Comment by Elsie19** [comment]
Please make this a draft until you are ready for review.

**Comment by Xdavius** [comment]
> Please make this a draft until you are ready for review.

Should be good with lazarus-deb merged

**Comment by Elsie19** [comment]
All the tests fail.

**Comment by Xdavius** [comment]
> All the tests fail.

I'm testing right now. lazarus-deb doesn't provide the default symlink... (No update-aternative, no postinstall in the Deb...)

And the package lazarus-deb doesn't support pacdeps so it doesn't download his deps... it's really a pain...

So... I need to fix this.

**Comment by Elsie19** [comment]
> And the package lazarus-deb doesn't support pacdeps so it doesn't download his deps... it's really a pain...

[We are working on that](https://github.com/pacstall/pacstall/pull/1062) for 5.0.0.

**Comment by Xdavius** [comment]
> > And the package lazarus-deb doesn't support pacdeps so it doesn't download his deps... it's really a pain...
> 
> [We are working on that](https://github.com/pacstall/pacstall/pull/1062) for 5.0.0.

I have found a solution, puting deps in order in pacdeps...

But, I have a new error with Lazarus set as dep (but I have no more depend on my test copy... I should reinstall a clean VM to continue, after the weekend now)

**Comment by Xdavius** [comment]
Hi.

I have modified my fork but, I see that if I modify it, update directly here. I don't want this but I don't know how to do. My branch is like synchonized to the PR.

**Pull request closed without merging**